phalcon/mvc/micro/lazyloader.zep

```diff
@@ -62,4 +62,9 @@ class LazyLoader
 		 */
 		return call_user_func_array([handler, method], arguments);
 	}
+
+	public function getDefinition()
+	{
+		return this->_definition;
+	}
```

Comment:
@sergeyklay can you link me to relevant documentation about that? I'm new to zephir :^)